medical/models/medical_patient.py

```diff
@@ -60,110 +34,79 @@ class MedicalPatient(orm.Model):
-                                              "Related Medical Centers",
-                                              readonly=True),
-        'age': fields.function(_get_age, type='char', string='Age',
-                               help="It shows the age of the patient "
-                                    "in years(y), months(m) and days(d).\n"
-                                    "If the patient has died, the age shown is"
-                                    "the age at time of death, the age "
-                                    "corresponding to the date on the death "
-                                    "certificate. It will show "
-                                    "also \"deceased\" on the field",
-                               multi=False),
-        'marital_status': fields.selection([('s', 'Single'),
-                                            ('m', 'Married'),
-                                            ('w', 'Widowed'),
-                                            ('d', 'Divorced'),
-                                            ('x', 'Separated'),
-                                            ('z', 'law marriage'), ],
-                                           string='Marital Status',
-                                           sort=False),
-        'deceased': fields.boolean(string='Deceased'),
-        'dod': fields.datetime(string='Date of Death'),
-        'identification_code': fields.char(size=256,
-                                           string='Internal '
-                                                  'Identification',
-                                           help='Patient Identifier '
-                                                'provided '
-                                                'by the Health '
-                                                'Center.Is not the '
-                                                'Social Security '
-                                                'Number'),
-        'active': fields.boolean('Active',
-                                 help="If unchecked, it will allow "
-                                      "you to hide the patient without "
-                                      "removing it."),
-    }
-    _defaults = {'is_patient': True, 'customer': True, 'active': True, }
-
-    def create(self, cr, uid, vals, context=None):
-        if not vals.get('identification_code', False):
-            sequence = unicode(
-                self.pool.get('ir.sequence').get(cr, uid, 'medical.patient'))
+                delta = relativedelta(now, dob)
+                deceased = ''
+            years_months_days = str(delta.years) + _('y ') + str(
+                delta.months) + _('m ') + str(delta.days) + _('d')\
+                + deceased
+        else:
+            years_months_days = _('No DoB !')
+        self.age = years_months_days
+
+    age = fields.Char(
+        string='Age', compute='_compute_age')
```

Comment:
`string=xxx` is not required in this case. https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#field 
Can you take into account this comment for the others fields?